src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -136,7 +136,7 @@ protected virtual void Dispose(bool disposing)
             OnPropertyChanged(nameof(IconBitmap));
             OnPropertyChanged(nameof(PrefixReserved));
 
-            var getVersionsTask = searchResultPackage.GetVersionsAsync();
+            var getVersionsTask = searchResultPackage.GetVersionsAsync(searchResultPackage.PackageLevel == PackageLevel.Transitive);
```

Comment:
I think we can compute if a package is transitive or not in `GetVersionsAsync` because the object `searchResultPackage` contains the required state in property `searchResultPackage.PackageLevel`.